naming/src/main/java/com/alibaba/nacos/naming/exception/ResponseExceptionHandler.java

```diff
@@ -30,7 +30,7 @@
 
     @ExceptionHandler(NacosException.class)
     private ResponseEntity<String> handleNacosException(NacosException e) {
-        Loggers.SRV_LOG.error("got exception. {}", e.getErrorMsg(), e);
+        Loggers.SRV_LOG.warn("got exception. {}", e.getErrorMsg(), e);
```

Comment:
This is the log triggered by all errors that throw NacoException, not just the 'serivce not found' error. So your modification is global and would influence other kinds of errors that throw NacoException.